### PR TITLE
Adds new TVL to popcorn

### PR DESCRIPTION
**Pull request by bhanusanghi**
Adds 3x Set Token to TVL, adds gelato Uni pools to pool2, and staking vault for gelato-uni lp tokens to tvl.

**NOTE**

1. The protocol is usually listed within 24 hours of merging the PR
2. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
3. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
4. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI

---

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/popcorn exports TVL: 
        
 
 
    arbitrum                  0
    bsc                       0
    bsc-staking               33.87 k
    arbitrum-staking          62.47 k
    ethereum-staking          2.74 M
    ethereum-pool2            832.72 k
    ethereum                  1.05 M
    polygon-pool2             22.10 k
    polygon-staking           1.19 M
    polygon                   12.99 k
    staking                   4.02 M
    pool2                     854.82 k
    
    total                    1.06 M

File: projects/popcorn/index.js [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
 function getTVL(chain = undefined) {
   return async (timestamp, block, chainBlocks) => {
     let balances = {};
     if (chain && chain === 'ethereum') {
-      // await addButterTVL(balances, timestamp, chainBlocks, chain)
-      await addButterV2TVL(balances, timestamp, chainBlocks, chain)
+      await addButterV2TVL(balances, timestamp, chainBlocks, chain);
+      await addThreeXTVL(balances, timestamp, chainBlocks, chain);
     }
+    await addStakingPoolsTVL(balances, timestamp, chainBlocks, chain)
```
Comment (g1nt0ki) [inline projects/popcorn/index.js:15]: hi, the rest of the code looks good, just this line, can this tvl be moved to pool2 instead?

**Comment by llamatester** [comment]
The adapter at projects/popcorn exports TVL: 
        
 
 
    polygon                   0
    bsc                       0
    arbitrum                  0
    bsc-staking               40.51 k
    ethereum-staking          2.74 M
    arbitrum-staking          62.02 k
    ethereum                  1.05 M
    ethereum-pool2            78.44 k
    polygon-staking           1.18 M
    polygon-pool2             11.38 k
    staking                   4.02 M
    pool2                     89.81 k
    
    total                    1.05 M

File: projects/popcorn/index.js (hunk 1 above)
Comment (bhanusanghi) [reply projects/popcorn/index.js:21]: Done

**Pull request merged**